src/plugins/telemetry/server/routes/telemetry_opt_in_stats.ts

```diff
@@ -40,10 +40,10 @@ export async function sendTelemetryOptInStatus(
   statsGetterConfig: StatsGetterConfig
 ) {
   const { optInStatusUrl, newOptInStatus, currentKibanaVersion } = config;
-  const optInStatus = await telemetryCollectionManager.getOptInStats(
+  const optInStatus = (await telemetryCollectionManager.getOptInStats(
     newOptInStatus,
     statsGetterConfig
-  );
+  )) as any;
```

Comment:
`getOptInStats` is typed. why do we have to cast it to `any` here?

Reply:
👍